src/client/Microsoft.Identity.Client/ApiConfig/AbstractAcquireTokenParameterBuilder.cs

```diff
@@ -122,6 +123,8 @@ public T WithExtraQueryParameters(string extraQueryParameters)
         }
 
         /// <summary>
+        /// Important: Use WithTenantId or WithTenantIdFromAuthority instead, or WithB2CAuthority for B2C authorities.

```

Comment:
> Use WithTenantId or WithTenantIdFromAuthority instead

Does this include ADFS and DSTS authortities?

Reply:
No, those are not supported scenarios